src/components/Button/horizontalScrollButton.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from './';
+import leftArrow from 'assets/leftArrow.svg';
+import rightArrow from 'assets/rightArrow.svg';
+import emptyLeft from 'assets/emptyLeft.svg';
+import emptyRight from 'assets/emptyRight.svg';
+
+const HorizontalScrollButtonWrapper = styled.div`
+	positon: relative;
+	border-radius: ${({ theme }) => theme.radius.md};
+`;
+
+const StyledIconButton = styled.img`
+	:hover {
+		cursor: pointer;
+	}
+`;
+
+const StyledHiddenIconButton = styled.img`
```

Comment:
We don't need this, instead we can pass in a prop (hidden) to StyedIconButton with if it is hidden or not. -- But, in this case we are going to delete this anyways. Add something like display: ${({ hidden }) => "none" : "block"}; to HorizontalScrollButton